docs/content.zh/docs/dev/table/materialized-table/statements.md

```diff
@@ -200,11 +200,11 @@ CREATE MATERIALIZED TABLE my_materialized_table
 CREATE MATERIALIZED TABLE my_materialized_table_continuous
     PARTITIONED BY (ds)
     WITH (
-        'format' = 'json',
+        'format' = 'debezium-json',
```

Comment:
I am curious. I look in the docs and some examples are `'format' = 'debezium-json' `an some are `'format' = 'json'.`. As you have changed this, even though this was not mentioned in the Jira, should all the formats be  'debezium-json' or are some validly `'format' = 'json'.`? It would be worth including mentioning in the docs what the supported formats are and the considerations around  `'debezium-json'` vs `'json'` if both are supported.

Reply:
@davidradl OK, I will check the value of 'format' property, i just found the sample sql in the Chinese version is different from that in the English version, and the Chinese version is incorrect, so I want to keep it consistent with the English version.

Reply:
@davidradl [debezium](https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/connectors/table/formats/debezium/) . The `testCreateMaterializedTableFailedInInContinuousMode()` test case in `MaterializedTableStatementITCase` is`'format' = 'json'`, so both are supported.